### PR TITLE
fix: verify sudo is present before using

### PR DESCRIPTION
**Pull request by troian**
run sudo in non-interactive mode
verify `service` command is available

### CODE REVIEW

File: pkg/hwinfo/hwinfo_nonwindows.go [hunk 1]
```diff
@@ -17,14 +17,14 @@ import (
...
 
-func isCommandAvailable(name string) bool {
-	cmd := exec.Command("/bin/sh", "-c", "command", "-v", name)
-	if err := cmd.Run(); err != nil {
-		return false
-	}
-
-	return true
-}
+// func isCommandAvailable(name string) bool {
```
Comment (martinlindhe) [inline pkg/hwinfo/hwinfo_nonwindows.go]: Should be deleted

Comment (troian) [reply pkg/hwinfo/hwinfo_nonwindows.go]: done

File: out.json [hunk 2]
```diff
@@ -0,0 +1,2560 @@
+{
```
Comment (martinlindhe) [inline out.json]: I think this file was added in accident ?

Comment (troian) [reply out.json]: done

**Pull request closed without merging**